examples/grpc_bidirectional_streaming/client/chat.pb.bal

```diff
@@ -15,8 +15,8 @@ public type ChatStub object {
         self.stub = navStub;
     }
 
-    function chat(typedesc listener, grpc:Headers... headers) returns (grpc:Client|error) {
```

Comment:
Please fix the file names.  It is not compliant with the convention we use. You can't use camel case for file name, but rather use "_" as separators only.

Reply:
Fixed da976c3